code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -1462,12 +1305,6 @@ func (sa *StorageAllocation) replaceBlobber(blobberID string, sc *StorageSmartCo
 				return actErr
 			}
 
-			blobber, err := sc.getBlobber(d.BlobberID, balances)
-			if err != nil {
-				return common.NewError("fini_alloc_failed",
-					"can't get blobber "+d.BlobberID+": "+err.Error())
-			}
```

Comment:
We need activation to remove code, though most likely this getting would not fail, but if it fail somehow, we will see error in some miners that is not updated, and no error in other miners with the latest version.